core/src/main/java/com/predic8/membrane/core/interceptor/oauth2/processors/DefaultEndpointProcessor.java

```diff
@@ -29,10 +39,20 @@ public boolean isResponsible(Exchange exc) {
     }
 
     @Override
-    public Outcome process(Exchange exc) throws Exception {
+    public Outcome process(Exchange exc) {
     	if (exc.getResponse() == null) {
-            exc.setResponse(OAuth2Util.createParameterizedJsonErrorResponse(jsonGen, "error", "invalid_request"));
-    	}
-        return Outcome.RETURN;
+            try {
+                exc.setResponse(OAuth2Util.createParameterizedJsonErrorResponse(jsonGen, "error", "invalid_request"));
+            } catch (IOException e) {
+                log.error(e.getMessage(), e);
```

Comment:
`e.getMessage()` is already printed, because `e` is passed to `log.error(...)` in the `Throwable` parameter.